framework/core/Vcs.pm

```diff
@@ -409,15 +409,17 @@ sub rev_date {
 # Helper subroutines
 
 #
-# Read commit-db and build cache
+# Read active-bugs csv and build cache
 #
 sub _build_db_cache {
     my $db = shift;
-    open (IN, "<$db") or die "Cannot open commit-db $db: $!";
+    open (IN, "<$db") or die "Cannot open active-bugs csv $db: $!";
     my $cache = {};
+    
+    my $header = <IN>;
     while (<IN>) {
         chomp;
-        /(\d+),([^,]+),([^,]+)/ or die "Corrupted commit-db!";
+        /(\d+),([^,]+),([^,]+)/ or die "Corrupted active-bugs csv!";
```

Comment:
While this does match, it is not corresponding to the expected format.

Reply:
Updated